tee-worker/client-api/parachain-api/prepare-build/interfaces/omniExecutor/definitions.ts

```diff
@@ -0,0 +1,96 @@
+export default {
```

Comment:
I am not familiar with this part. Will we add tests later, or do we already have tests for these definitions?

Reply:
We will use these definitions to write some tests in the omni-executor